tests/Makefile.am

```diff
@@ -45,6 +45,7 @@ testsuite_sources = \
 	testsuite.src/syn_set.at \
 	testsuite.src/syn_subscripts.at \
 	testsuite.src/syn_value.at \
+	testsuite.src/syn_format.at \
```

Comment:
A new file is fine, but if we do this, then the existing tests from syn_misc.at and configuration.at (the later wasn't a very good place for it in any case) should be moved there, too (just `grep -R tests/testuite.src "fformat"` - most of those should be moved).

As an alternative if you feel that's too much for the available time: stick the new test to the ~~trash~~ catch-all syn_misc.at, please.

Reply:
Ok, indeed, I moved the test back to `syn_misc.at`, given the lack of time.

I modified the `texi` documentation too.

And I don't know why, but I cannot find a button to switch the PR back from the draft mode... Maybe it's because you have requested changes ?

Reply:
Hm, so you don't have that at the PRs bottom?

![PR state](https://user-images.githubusercontent.com/6699539/215595317-6fd9f5e3-9d08-4153-9eee-9d513d51b093.png)